themes/SuiteP/css/suitep-base/listview.scss

```diff
@@ -941,7 +941,7 @@ a.listViewTdToolsS1 {
 
 #selectLinkTop > .sugar_action_button > .subnav a {
   background-color: $list-view-action-menu-bg;
-  color: $default-btn-color;
+  color: $white;
```

Comment:
Are these meant to be white on all the other themes like Dawn/Day/Dusk? since this is changing the base it will impact every theme.

Reply:
Hi dillon i have cross verified the all the themes with built command seperatly. Its working fine without impacting the color changes.

Reply:
So looking at Day we have:

```
$default-btn-color: $color-51;
$color-51: #F5F5F5;
```

And then in Dusk we have:

```
$default-btn-color: $color-3;
$color-9: #D3D3D3;
```

These are close enough to white that it's pretty unnoticeable, however I think we lose some ability to customise by changing the variables here to ```$white```. Would it not make sense to keep them as ```$default-btn-color``` and then just change the variable assignment specifically in the Noon theme? or would that have a larger impact?